plugins/modules/user_role_workflow_manager.py

```diff
@@ -963,6 +963,33 @@ def validate_string_field(self, field_value, regex, error_message, error_message
         if field_value and not regex.match(field_value):
             error_messages.append(error_message)
 
+    def validate_password(self, password, error_messages):
+        """
+        Validate password from password and append error message if it does not match the criteria.
+        """
+
```

Comment:
if not password:
then log a message and return..

Reply:
Updated the code accordingly